### PR TITLE
Update locality records

### PR DESCRIPTION
**Pull request by stepps00**
This PR adds polygons to the locality of Brest. The existing point geometries were stored as alt files, and existing label centroids were kept as-is.

All of the locality points that fall within the new locality polygons look to be neighbourhoods, so they were superseded into new neighbourhood records (rather than flagged as funky). This PR needs to be PIP'd once approved.

**Number of records updated**: 28

### CODE REVIEW

**Comment by stepps00** [comment]
Screenshot 2023-06-28 at 3 29 12 PM

File: data/186/399/110/1/1863991101.geojson [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        }
+    ],
+    "wof:id":1863991101,
+    "wof:lastmodified":1687990137,
+    "wof:name":"Gershony",
+    "wof:parent_id":85632395,
+    "wof:placetype":"neighbourhood",
+    "wof:repo":"whosonfirst-data-admin-by",
+    "wof:superseded_by":[],
+    "wof:supersedes":[],
```
Comment (nvkelso) [inline data/186/399/110/1/1863991101.geojson:65]: Needs supersedes ID listed, please

**Review by nvkelso — CHANGES_REQUESTED** [review]
Same supersedes comment as the other PRs (for each record, I only noted 1 example).

**Review by nvkelso — APPROVED** [review]
Thanks!

**Pull request merged**